assets/keymaps/vim.json

```diff
@@ -335,22 +335,101 @@
     }
   },
   {
-    "context": "vim_mode == helix_normal",
+    "context": "vim_mode == helix_normal && !menu",
     "bindings": {
+      "escape": "editor::Cancel",
+      "ctrl-[": "editor::Cancel",
+      ":": "command_palette::Toggle",
+      "shift-d": "vim::DeleteToEndOfLine",
+      "shift-j": "vim::JoinLines",
+      "y": "editor::Copy",
+      "shift-y": "vim::YankLine",
       "i": "vim::InsertBefore",
+      "shift-i": "vim::InsertFirstNonWhitespace",
       "a": "vim::InsertAfter",
-      "d": "vim::HelixDelete",
-      "w": "vim::NextWordStart",
-      "e": "vim::NextWordEnd",
-      "b": "vim::PreviousWordStart",
+      "shift-a": "vim::InsertEndOfLine",
+      "o": "vim::InsertLineBelow",
+      "shift-o": "vim::InsertLineAbove",
+      "~": "vim::ChangeCase",
+      "ctrl-a": "vim::Increment",
+      "ctrl-x": "vim::Decrement",
+      "p": "vim::Paste",
+      "shift-p": ["vim::Paste", { "before": true }],
+      "u": "vim::Undo",
+      "ctrl-r": "vim::Redo",
+      "r": "vim::PushReplace",
+      "s": "vim::Substitute",
+      "shift-s": "vim::SubstituteLine",
+      ">": "vim::Indent",
+      "<": "vim::Outdent",
+      "=": "vim::AutoIndent",
+      "g u": "vim::PushLowercase",
+      "g shift-u": "vim::PushUppercase",
+      "g ~": "vim::PushOppositeCase",
+      "\"": "vim::PushRegister",
+      "g q": "vim::PushRewrap",
+      "g w": "vim::PushRewrap",
+      "ctrl-pagedown": "pane::ActivateNextItem",
+      "ctrl-pageup": "pane::ActivatePreviousItem",
+      "insert": "vim::InsertBefore",
+      // tree-sitter related commands
+      "[ x": "editor::SelectLargerSyntaxNode",
+      "] x": "editor::SelectSmallerSyntaxNode",
```

Comment:
This has already been closed, but I would like to understand why alt-o and alt-i aren't used as in helix for Seleting a larger tree-sitter sytnax node. Are they clashing with something else?

Reply:
We were just copying these from @jneem's config (which came from https://github.com/infogulch/zed-helix-keymap).

We could add `alt-o` and `alt-i` as aliases if those are the helix bindings - please do send a PR!

FWIW We generally don't use `alt-X` for keybindings because they don't transfer to non-US keyboards very well; but I'm comfortable copying helix's behavior if you're in helix_normal mode.

Reply:
> We could add alt-o and alt-i as aliases if those are the helix bindings - please do send a PR!

I will gladly do so as soon as I have some time this week